tsconfig.json

```diff
@@ -0,0 +1,40 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "newLine": "LF",
+    "target": "es6",
+    "importHelpers": true,
+    "moduleResolution": "node",
+    "noImplicitReturns": true,
+    "outDir": "dist",
+    "allowJs": false,
+    "noUnusedLocals":true,
+    "strict": true,
+    "strictPropertyInitialization": true,
+    "strictNullChecks": true,
+    "declaration": true,
+    "declarationDir": "./typings",
+    "lib": [
```

Comment:
do you actually need all of these? if you're going to output es6 then i would expect this to be something like ["es6", "dom"] (assuming you actually need dom APIs, which you won't have in node)

Reply:
One of the goals is to make this sdk isomorphic. Hence kept them for now. Can remove it as it is not needed at the moment.

Reply:
if you want isomorphic then you can just exclude "lib" altogether. the default lib for any given target is the set of isomorphic definitions. then "dom" can be added back in if you need it (typically due to dependencies that pull it in)